src/adapters.coffee

```diff
@@ -157,11 +175,11 @@ class Adapters
       throw 'ga not defined. Please make sure your Universal analytics is set up correctly' if typeof ga isnt 'function'
       ga('send', 'event', category, action, label, {'nonInteraction': 1})
 
-    @experiment_start: (experiment_name, variant) =>
-      @_track(@namespace, "#{experiment_name} | #{variant}", 'Visitors')
+    @experiment_start: (experiment, variant) =>
+      @_track(@namespace, "#{experiment.name} | #{variant}", 'Visitors')
```

Comment:
not in this tracking adapter... the tracking adapters don't share any code and can implement internally differently.